### PR TITLE
flx: Remove deprecated KeyPair class.

### PR DESCRIPTION
**Pull request by mpcomplete**

### CODE REVIEW

File: sky/packages/workbench/pubspec.yaml [hunk 1]
```diff
@@ -6,7 +6,8 @@ homepage: https://github.com/flutter/engine/tree/master/sky/packages/workbench
 dependencies:
   flutter: 
     '0.0.16'
-  sky_tools: any
+  sky_tools:
+    '0.0.34'
```
Comment (Hixie) [inline sky/packages/workbench/pubspec.yaml]: The flutter dependency should take care of this.
If you need this to be pinned as well, though, make sure to update the rev-flutter.dart script.

**Comment by Hixie** [comment]
LGTM

**Comment by mpcomplete** [comment]
OK, I updated the sky_tools version pointed to by the sky package.

File: sky/packages/sky/pubspec.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ dependencies:
     '0.0.46'
   sky_services: 
     '0.0.46'
-  sky_tools: '>=0.0.33 <0.1.0'
+  sky_tools: '>=0.0.34 <0.1.0'
```
Comment (abarth) [inline sky/packages/sky/pubspec.yaml]: I just updated this to 35.

**Pull request merged**